### PR TITLE
Automapper Profiles

### PR DESCRIPTION
**Pull request by james-hollinger**
Organize the AutoMapper profiles before undertaking fixing the EnrolleeViewModel

### CODE REVIEW

**Review by mtpultz — APPROVED** [review]
MTGL

File: prime-dotnet-webapi/ViewModels/Agreements/_AgreementMappingProfile.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+using AutoMapper;
+
+using Prime.Models;
+
+namespace Prime.ViewModels.Profiles
+{
+    public class AgreementMappingProfile : Profile
```
Comment (johnnyl6) [inline prime-dotnet-webapi/ViewModels/Agreements/_AgreementMappingProfile.cs:7]: any benefit using underscore prefix for the class filename like `_AgreementMappingProfile.cs`?

**Review by hogmann — APPROVED** [review]
Looks Good To Marty

File: prime-dotnet-webapi/ViewModels/Agreements/_AgreementMappingProfile.cs (hunk 1 above)
Comment (james-hollinger) [reply prime-dotnet-webapi/ViewModels/Agreements/_AgreementMappingProfile.cs:7]: Putting the underscore makes the profile bubble up to the top of the file list, making it easier to find. 
image

vs

image

**Pull request merged**